relational-query-HDBC/src/Database/HDBC/Query/TH.hs

```diff
@@ -174,14 +174,14 @@ defineTableFromDB connect driver tbl scm = tableAlongWithSchema connect driver t
 inlineVerifiedQuery :: IConnection conn
                     => IO conn      -- ^ Connect action to system catalog database
                     -> Name         -- ^ Top-level variable name which has 'Relation' type
-                    -> Relation p r -- ^ Object which has 'Relation' type
+                    -> Relation p r -- ^ Object which has 'Relation' type igrep TODO: disable placeholders?
```

Comment:
I misunderstood that this function runs `Relation` at compile-time.
But actually not. So the placeholder values are not necessary.